server/app/services/question/types/QuestionDefinition.java

```diff
@@ -75,10 +75,14 @@ public final Optional<Instant> getLastModifiedTime() {
     return config.lastModifiedTime();
   }
 
+  public final String getQuestionNameKey() {
+    return config.name().replaceAll("[^a-zA-Z ]", "").replaceAll("\\s", "_");
+  }
+
```

Comment:
Extracting out `getQuestionNameKey()` allows us to see the lowercase underscore version of a question name, in the API docs, such as `sample_name_question`, which is also the key for the API Response preview.